Shared/Samples/Download preplanned map area/DownloadPreplannedMapAreaView.Model.swift

```diff
@@ -58,6 +58,12 @@ extension DownloadPreplannedMapAreaView {
         /// A Boolean value indicating if the offline content can be deleted.
         @Published private(set) var canRemoveDownloadedMaps = false
         
+        /// A Boolean value indicating whether to select a map.
+        @Published var isShowingSelectMapView = false
```

Comment:
> The "Done" button now sets the selection value, but to do so the value had to be moved from to view to the model which is not ideal.

I don't think this problem should be fixed in the view level. It would be better addressed at the root, in `View+Sheet.swift`

Reply:
I fixed the dismiss sheet bug by adding back the logic from [this](https://github.com/Esri/arcgis-maps-sdk-swift-samples/pull/42/files#diff-de5737b47911aa7de83a8916aeab221a5d7c5f73823ba283aafd21360deae875R304) PR. Do you know why that logic was removed? It's necessary to set the sheet's `isSelected` property to `false` when dismissing the sheet.

Reply:
I'm not sure. Maybe it fell through the cracks and wasn't merged when the sample was refactored by Ryan.